src/routes/(infoPages)/+layout.svelte

```diff
@@ -16,20 +19,21 @@
 </div>
 <!-- Top Tier pages have a slightly different layout, with a hero image and title above the rest of the layout. -->
 {#if __typename === "TopTier"}
-  {#if topTierPage?.heroImage}
-    <!-- TODO: Update this to use a proper Hero/Image Component
-       (pending merge of https://github.com/la-ldaf/ldaf-site/pull/279) -->
-    <div
-      style={`
-        background-image:url(${topTierPage.heroImage?.imageSource?.url});
-        background-size: cover;
-        background-position: center;
-        background-repeat: no-repeat;
-        height: 33vh;
-        width: 100%;
-      `}
+  {#if topTierPage?.heroImage?.imageSource}
+    <Image
+      class="top-tier-page-hero-image"
+      src={topTierPage.heroImage.imageSource.url}
+      alt={topTierPage.heroImage.imageSource.description}
+      sources={getSources}
+      width={topTierPage.heroImage.imageSource.width}
+      height={topTierPage.heroImage.imageSource.height}
+      sizeType="full-bleed"
+      blurhash={topTierPage.heroImage.imageSource.blurhash}
+      fit={true}
+      preserveAspectRatio={false}
+      loading="eager"
```

Comment:
Most of these look good, but the one for `About LDAF` seems to be failing to load on larger screen sizes: https://ldaf-git-benaiah-use-image-component-instead-of-img-tags-ldaf.vercel.app/about

Might be an issue with the image itself?

Reply:
It's another case of us trying to convert to AVIF when the image is over 9 megapixels. Just pushed a fix.